### ISSUE TITLE
Handle hour and minute timescales

### ISSUE THREAD

**Issue by schoinh** [allen-cell-animated/simularium-website#176]
## Use Case

For some trajectories that have hours-long timescales (COVID one for example), it would be nice to be able to support hours and minutes

## Solution

Ideally display hours and minutes as something like HR:MM

### Comments from Jira:
![image](https://user-images.githubusercontent.com/12690133/124646413-74e3d800-de49-11eb-9efc-c641ce0355af.png)

**Comment by blairlyons** [comment]
yeah now that the user provides the time units, I think if the user-provided units are min or hours, instead of displaying their fractions as a decimal (e.g. 2.X ms, 5.X days), display them as :XX in base 60. So if units are hours, display HH:MM hrs, if units are minutes, display MM:SS min.
